src/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -1059,23 +1063,25 @@ private void notifyListeners() {
 
 class UnitPanel extends JPanel {
   private static final long serialVersionUID = 1509643150038705671L;
+  private final IUIContext m_context;
   private final UnitCategory m_category;
   private final ScrollableTextField m_textField;
+  private final GameData m_data;
   private final ListenerList<WidgetChangedListener> m_listeners = new ListenerList<>();
+  private final ScrollableTextFieldListener m_listenerTextField = new ScrollableTextFieldListener() {
+    @Override
+    public void changedValue(final ScrollableTextField field) {
+      notifyListeners();
+    }
+  };
```

Comment:
I do not think so,  ScrollableTextFieldListener not a Functional interface. We could create a wrapper to make it so, but meh..